assets/lokomotive-kubernetes/bootkube/resources/charts/lokomotive/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webhook-admission
+  labels:
+    name: webhook-admission
```

Comment:
Assuming this will be installed in the `kube-system` namespace, I wonder if we should mingle our workloads with the control plane. `kube-system` namespace, with its nature, has lot of permissions. To isolate this workload I would recomment to move this workload to its own namespace like `lokomotive-system` or just `lokomotive`.

Reply:
I'm also towards `lokomotive-system`, though this seems like an important decision.

Reply:
+1 to a lokomotive specific namespace.